mimix1050rt: gPTP build fix

agansari opened the pull request:
Sets iMX RT's cmake and fixup to enable gPTP in MCUX_ETH.

Signed-off-by: Andrei Gansari <andrei.gansari@nxp.com>

Fixes an issue observed by @jeremy-e-mills in #11586

dbkinder reviewed (approved):
+1 for doc change

agansari commented:
@MaureenHelm please have a look at this?

MaureenHelm commented on samples/net/gptp/README.rst:
```diff
@@ -33,7 +33,7 @@ Follow these steps to build the gPTP sample application:
    :zephyr-app: samples/net/gptp
    :board: <board to use>
-   :conf: prj.conf
+   :conf: prj_base.conf
```
Why are you changing this?

jukkar replied:
Good catch, but that should be placed in separate commit as it is unrelated to mimix1050rt support.

agansari committed:
mimix1050rt: gPTP build fix

Sets iMX RT's cmake and fixup to enable gPTP in MCUX_ETH.

Signed-off-by: Andrei Gansari <andrei.gansari@nxp.com>

agansari committed:
samples: minor documentation fixes

DHCPv4 client typo: '/etc/dhcpd/dhcp.conf'
gPTP Sample -DCONF_FILE=prj.conf (filename changed)

Signed-off-by: Andrei Gansari <andrei.gansari@nxp.com>

agansari reviewed (commented):
@jukkar @MaureenHelm split documentation update to a separate commit
I added another minor documentation fix to close #11531

jukkar reviewed (approved):
LGTM
